docs/demo.mdx

```diff
@@ -0,0 +1,506 @@
+import HomepageHeader from '@site/src/components/Homepage/Header/HomepageHeader';
+
+import '@site/src/css/custom.css';
+import '@site/node_modules/infima/dist/css/default/default.css';
+
+import FlavorizedCodeBlock from '@site/src/components/FlavorizedCodeBlock';
+
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# H1 heading
+
+## H2 heading
+
+### H3 heading
+
+#### H4 heading
+
+##### H5 heading
+
+###### H6 heading
+
+## Admonitions:
+
+:::caution CAUTION
+
+This is a CAUTION admonition with a custom background color, and a border color.
+
+:::
+
+:::danger DANGER
+
+This is a DANGER admonition with a custom background color, and a border color.
+
+:::
+
+:::info INFO
+
+This is an INFO admonition with a custom background color, and a border color.
+
+:::
+
+:::success TIP
+
+This is a TIP admonition with a custom background color, and a border color.
+
+:::
+
+:::note NOTE
+
+This is a NOTE admonition with a custom background color, and a border color.
+
+:::
+
+#### _BELOW is an admonition with code span and a link:_
```

Comment:
Not critical, but in my opinion the variations of each type should be next to each other. This helps to better understand the changes when working on the color of a specific admonition type.

- `:::tip`
- `:::tip` + links and code span
- `:::tip` + long content (more than one line) + other elements
- `:::note`
- `:::note` + links and code span

Reply:
also you don't have to duplicate the admonition with long content for every kind if it only differs by the block colors, that's enough to have it in for one type

Reply:
we still need to present each of them with its type-specific variation (for instance, content with one line + link + code span)

Reply:
Thank you! I worked on this and per Yarik's comments, just used the type-specific variations as the displayed admonitions on the page. Resolved.